src/solver/conv_asm_1x1u.cpp

```diff
@@ -667,27 +666,26 @@ ConvSolution ConvAsm1x1U::GetSolution(const ConvolutionContext& params,
     GenerateClangDefsym(options, "ROCM_METADATA_VERSION", params.rmv.UseV3() ? 5 : 4);
 
     const PerformanceConfigConvAsm1x1U* pcfg = &config;
+
+    // Try to load config from environment variable
```

Comment:
[Style][Recommendation] If the code is not clear enough, then it may worth factoring it out to some entity (most likely, a function in this specific case) with some self-explanatory name. Adding a comment should be considered as a last resort.

Reply:
@averinevg [Note] When I provide a "recommendation", the decision is totally up to you :wink: